src/libraries/pkg/test/testPackages.proj

```diff
@@ -3,19 +3,26 @@
 
   <ItemGroup>
     <PackagesToTest Condition="'$(PackagesToTest)' != ''" Include="$(PackagesToTest)" />
-    <!-- transport package not expected to be closure complete  -->
-    <ExcludePackages Include="Microsoft.AspNetCore.Internal.Transport" />
-    <ExcludePackages Include="$(ExcludePackages)" />
 
     <PackageReports Condition="'@(PackagesToTest)' == ''" Include="$(PackageReportDir)*.json" Exclude="@(ExcludePackages->'$(PackageReportDir)%(Identity).json')"/>
     <PackageReports Condition="'@(PackagesToTest)' != ''" Include="@(PackagesToTest->'$(PackageReportDir)%(Identity).json')" />
 
     <!-- support override via commandline -->
     <RuntimesToInclude Condition="'$(RuntimesToInclude)' != ''" Include="$(RuntimesToInclude)" />
     <TargetFrameworksToInclude Condition="'$(TargetFrameworksToInclude)' != ''" Include="$(TargetFrameworksToInclude)" />
+    <SymbolsPackages Include="$(LibrariesAllConfigShippingPackagesDir)*.symbols.nupkg" />
+    <AllPackagesFilePath Include="$(LibrariesAllConfigShippingPackagesDir)*.nupkg" Exclude="@(SymbolsPackages)" Condition="'$(SymbolsPackages)' != ''">
+      <PackageName>$([System.IO.Path]::GetFileNameWithoutExtension('%(Identity)').Substring(0, $([System.IO.Path]::GetFileNameWithoutExtension('%(Identity)').IndexOf('.$(MajorVersion)'))))</PackageName>   
```

Comment:
This is still a bit of hack, not everything we build has the same major version.  Could you filter later in the build?  Perhaps after you've read the package?

Reply:
we unified everything to the same packageVersion during the last release

Reply:
I would like to avoid us depending on the same major version across all packages.

Reply:
If the intent of this code is to just get the package name without its version, why not use a simple regex?

Reply:
I could image that the indexof would even work without a regex if you just want to trim out everything after ".\d" (without a regex \d would be "0123456789".

Reply:
i ended up using the regex ```[a-zA-z\.]*\.``` but regex was a good suggestion